redbot/cogs/admin/admin.py

```diff
@@ -200,6 +202,7 @@ async def _removerole(
             await ctx.send(_(NEED_MANAGE_ROLES))
             return
         try:
+            reason = get_audit_reason(ctx.author)
             await member.remove_roles(role)
```

Comment:
```suggestion
            await member.remove_roles(role, reason=reason)
```

Are you not missing this?

Reply:
ah yeah totally missed that one lol my bad, ty for the update